src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -147,18 +148,19 @@ export const CloudEnvironmentModal = ({
 
   // We assume here that button disabling is working properly, so the only thing to check is whether it's an app or the current (assumed to be existing) runtime
   const startApp = toolLabel => Utils.cond([isToolAnApp(toolLabel), () => {
-    const { googleProject, appName } = currentApp(toolLabel)
+    const { appName, cloudContext } = currentApp(toolLabel)
     executeAndRefresh(toolLabel,
-      Ajax().Apps.app(googleProject, appName).resume())
+      //TODO: Potential issue in leonardo needs to be fixed.
```

Comment:
What is the TODO here referencing?

Reply:
That there may be an issue in leo that was the root of it - however, I'm planning to remove the TODO because we want to move to using cloudContext over googleProject regardless.